README.md

```diff
@@ -168,6 +168,14 @@ By default, `snyk-monitor` uses an emptyDir for temporary storage. If you prefer
 The PVC's name defaults to `snyk-monitor-pvc`. If you prefer to override this, then use the following value:
 * `pvc.name`
 
+## Using custom CA certificate
+
+You can use a custom CA certificate by setting the following variable in the `ConfigMap`:
+
+```yaml
+extraCaCerts: /srv/app/certs/ca.pem
```

Comment:
This hasn't been added to the YAML files intentionally since we want to stop supporting that mode of installation, so only the Helm chart and public docs could be updated 🙏

Reply:
so we should never update on this readme?

Reply:
I guess only when it is also added to `snyk-monitor-deployment.yaml`, for this specific feature we added it only to the Helm chart 😕